recipe/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+mesa-libGL
```

Comment:
@dpryan79 should this be sufficient? I was including the whole set from [the KB note](https://conda-forge.org/docs/maintainer/knowledge_base.html#libgl) on other feedstocks. I.e., 

```
mesa-libGL
mesa-dri-drivers
libselinux
libXdamage
libXxf86vm
libXext
```

Curious if I should have done this instead.